tests/integration_tests/databases/api_tests.py

```diff
@@ -1984,6 +1984,7 @@ def test_import_database(self):
         dataset.owners = []
         database.owners = []
         db.session.delete(dataset)
+        db.session.commit()
```

Comment:
I think there's a quirk with how the owner relationships are defined—in terms of cascading—and without committing the dataset deletion, SQLAlchemy complains that the owners are stale.